05-Interfaces/string_converter.c

```diff
@@ -35,6 +43,18 @@ static void conv_uppercase(char *str, size_t count)
 
 }
 
+static void conv_lowercase(char *str, size_t count)
+{
+	int i;
+
+	for (i = 1; i < count; i++) {
+		if (str[i] >= 'A' && str[i] <= 'Z') {
+			str[i] += 32;
```

Comment:
what is 32?

Reply:
This is a magic number, this number is the result of an arithmetic operation " 'A' - 'a' ". Fixed, used ASCII.